lib/ecto/adapters/myxql/connection.ex

```diff
@@ -1080,6 +1091,9 @@ if Code.ensure_loaded?(MyXQL) do
       [?`, name, ?`]
     end
 
+    defp format_to_sql(:json), do: "FORMAT=JSON"
```

Comment:
```suggestion
    defp format_to_sql(:map), do: "FORMAT=JSON"
```
If we are able to decode the result as per my other comment